collections/_article/dev-snapshot-godot-4-2-dev-4.md

```diff
@@ -0,0 +1,200 @@
+
+We're slightly behind our internal release schedule on that build, but only because we had some unplanned maintenance of the build server. Otherwise the overall state of the `master` branch seems pretty good and on track for reaching the beta stage at the start of October, and stable in early November 2023.
+
+This snapshot brings a lot of goodies! Here are some of the highlights, with a bigger list available [below](#whats-new):
+
+- The initial support for C# on Android merged for dev 3 ([GH-73257](https://github.com/godotengine/godot/pull/73257)) is now ready for mass testing! Official export templates are provided in the .NET ("Mono") build, so you can start exporting your C# projects to Android and give us feedback on what works and what doesn't. Keep in mind that this feature requires using .NET 7.0 as the target framework, which you can set in the `.csproj` file with `<TargetFramework>net7.0</TargetFramework>. It's still a work in progress, with caveats outlined in the PR, notably we already know that it's not working on the arm32 architecture.
+
+- A number of rendering bugfixes and improvements were merged for this snapshot, including:
+  * Motion vectors for skeletons/blend shapes and particles ([GH-80618](https://github.com/godotengine/godot/pull/80618) and [GH-80688](https://github.com/godotengine/godot/pull/80688)). This is the last bit of work necessary before we implement AMD's FSR 2.2 into Godot. On that note, there is already a PR for FSR 2.2 support ([GH-81197](https://github.com/godotengine/godot/pull/81197)), so please test it out and give us your feedback.
+  * Fixing a crash caused by having more than 204 Lights/Decals/ReflectionProbes in a scene at once ([GH-80845](https://github.com/godotengine/godot/pull/80845)). This bug stopped users from fully flexing the strength of the Forward+ renderer as it limited users to a combined 204 OmniLight3Ds/SpotLight3Ds/Decals/ReflectionProbes (by default). Now users are free to use up to 512 of each in the default configuration (this limit can be increased).
+  * Additionally, rendering contributors have been busy removing barriers and preparing to implement more items from the list of [rendering priorities](https://godotengine.org/article/rendering-priorities-july-2023).
+
+- A solution was finally merged ([GH-80859](https://github.com/godotengine/godot/pull/80859)) for an infamous input bug affecting the typical input action combinations used for character movement (e.g. WASD + D-Pad). We're expecting potential regressions (and already found one, with a pending fix as [GH-81170](https://github.com/godotengine/godot/pull/81170)), so please test it thoroughly and report any issue.
```

Comment:
Should this be "WASD or D-Pad"? Or "WASD/D-Pad"? I'm not familiar with the bug so maybe "WASD + D-Pad" is correct, just wanted to check.

Reply:
It's both WASD and D-Pad at the same time.